t/JrDevJobs.t

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+use_ok('DDG::Spice::JrDevJobs');
+
```

Comment:
Whoops, looks like you don't have any actual tests here to ensure legitimate queries make the correct API calls. Please add some real tests